packages/core/src/utils/env.ts

```diff
@@ -15,7 +15,7 @@
 
 declare const __SENTRY_BROWSER_BUNDLE__: boolean | undefined;
 
-export type SdkSource = 'npm' | 'cdn' | 'loader';
+export type SdkSource = 'npm' | 'cdn' | 'loader' | 'aws';
```

Comment:
Any opinions on the source prefix? `aws`, `aws-layer`, `lambda-layer`, `aws-lambda-layer`?